autoshop/api/v1/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+from core.models import Profile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Profile
+        fields = '__all__'
```

Comment:
All important data is not present in Profile model. Username, email, password etc are present in Django's default User model and Profile model only has User ID associated with it other than profile information.